verticapy/tests/utilities/test_utilities.py

```diff
@@ -12,49 +12,276 @@
+        )
+        base.cursor.execute(
+            "INSERT INTO drop_data(transportation, gender, \"owned cars\", cost, income) VALUES ('Bus', 'Male', 1, 'Cheap', 'Med')"
+        )
+        base.cursor.execute(
+            "INSERT INTO drop_data(transportation, gender, \"owned cars\", cost, income) VALUES ('Train', 'Female', 1, 'Cheap', 'Med')"
+        )
+        base.cursor.execute(
+            "INSERT INTO drop_data(transportation, gender, \"owned cars\", cost, income) VALUES ('Bus', 'Female', 0, 'Cheap', 'Low')"
+        )
+        base.cursor.execute(
+            "INSERT INTO drop_data(transportation, gender, \"owned cars\", cost, income) VALUES ('Bus', 'Male', 1, 'Cheap', 'Med')"
+        )
+        base.cursor.execute(
+            "INSERT INTO drop_data(transportation, gender, \"owned cars\", cost, income) VALUES ('Train', 'Male', 0, 'Standard', 'Med')"
+        )
+        base.cursor.execute(
+            "INSERT INTO drop_data(transportation, gender, \"owned cars\", cost, income) VALUES ('Train', 'Female', 1, 'Standard', 'Med')"
+        )
+        base.cursor.execute(
+            "INSERT INTO drop_data(transportation, gender, \"owned cars\", cost, income) VALUES ('Car', 'Female', 1, 'Expensive', 'Hig')"
+        )
+        base.cursor.execute(
+            "INSERT INTO drop_data(transportation, gender, \"owned cars\", cost, income) VALUES ('Car', 'Male', 2, 'Expensive', 'Med')"
+        )
+        base.cursor.execute(
+            "INSERT INTO drop_data(transportation, gender, \"owned cars\", cost, income) VALUES ('Car', 'Female', 2, 'Expensive', 'Hig')"
+        )
+        base.cursor.execute("COMMIT")
+        # table
+        base.cursor.execute("DROP TABLE IF EXISTS public.verticapy_table_test")
+        base.cursor.execute("CREATE TABLE verticapy_table_test AS SELECT 1;")
+        drop("verticapy_table_test", base.cursor)
+        base.cursor.execute(
+            "SELECT table_name FROM columns WHERE table_name = 'verticapy_table_test' GROUP BY 1;"
+        )
+        result = base.cursor.fetchall()
+        assert result == []
+        # view
+        base.cursor.execute("DROP VIEW IF EXISTS public.verticapy_view_test")
+        base.cursor.execute("CREATE VIEW verticapy_view_test AS SELECT 1;")
+        drop("verticapy_view_test", base.cursor)
+        base.cursor.execute(
+            "SELECT table_name FROM view_columns WHERE table_name = 'verticapy_view_test' GROUP BY 1;"
+        )
+        result = base.cursor.fetchall()
+        assert result == []
+        # text index
+        base.cursor.execute(
+            "CREATE TEXT INDEX drop_index ON drop_data (id, transportation);"
+        )
+        drop("drop_index", base.cursor)
+        try:
+            base.cursor.execute("SELECT * FROM drop_index;")
+            fail = False
+        except:
+            fail = True
+        assert fail
```

Comment:
Pytest has a standard way of asserting exceptions:
https://docs.pytest.org/en/reorganize-docs/new-docs/user/pytest_raises.html
https://docs.pytest.org/en/stable/reference.html#pytest-raises

Reply:
Corrected